plugins/utils_operators/agol_operators.py

```diff
@@ -186,6 +190,7 @@ def get_fields(self, query_url):
         params = "&".join([ f"{k}={v}" for k,v in query_string_params.items() ])
         url = "https://" + "/".join([ url_part for url_part in f"{query_url}/query?{params}".replace("https://", "").split("/") if url_part ])
         res = requests.get(url)
+        logging.info("Try and get fields: ", res)
```

Comment:
@garyqi-to this line throws an error - `logging.info` only takes one string argument, and here you've fed it a string and a `requests.Response` object

Reply:
Please remove agol_operator.py from this PR as well. not suppose to be. My bad